### ISSUE TITLE
Threads of deleted contact are green / unread after every login

### ISSUE THREAD

**Issue by httng** [PeerioTechnologies/peerio-client#108]
I have two threads with a now removed contact that are shown as unread (bold font and green indicator) every time I login, even though I opened and read them. Opening them will remove the green indicator, but next time I log in it will be the same again.

Possibly relevant for reproduction: In both threads, the deleted contact attempted to send a message after the relation was broken (by me deleting him as a contact). Both threads have a 'Your contact attempted to send a message, but it could not be properly received by your account.' message in them.

**Comment by flohdot** [comment]
Thanks for the detailed report! I will look into this.

**Comment by httng** [comment]
To be more complete, a comparison of both threads:

Thread A was started by DeletedContact.
- Contains some messages back and forth
- Ends with a failed attempt to reach me after I deleted the contact 'Your contact attempted to send a message, but it could not be properly received by your account.'
- On top of the thread, both me and DeletedContact are still listed as participants. (perhaps this means DeletedContact did not remove this thread? If so, that might be an issue on it's own).

Thread B was started by me.
- Starts with my message to DeletedContact that I would delete him for testing purposes
- Right after that I deleted DeletedContact
- Then some more messages from me (never delivered, but shown in thread)
- A failed attempt from DeletedContact to reach me after I deleted the contact 'Your contact attempted to send a message, but it could not be properly received by your account.'
- On top of the thread, it shows only my name, and in red DeletedContact as a former participant.
